app/models/agents/rss_agent.rb

```diff
@@ -122,5 +125,22 @@ def check_and_track(entry_id)
         true
       end
     end
+
+    def feed_to_events(feed)
+      feed.entries.map { |entry|
+        Event.new(payload: {
```

Comment:
Does `create_event` accept an event object and assign the Agent and User in master right now?